src/actions/index.js

```diff
@@ -19,6 +24,12 @@ export const handleCreateButton = (isCreateButtonClicked) => {
   };
 };
 
+export const deleteTodo = (id) => {
+  return {
+    type: DELETE_TODO,
+    id,
```

Comment:
see https://redux.js.org/tutorials/essentials/part-4-using-data#preparing-action-payloads

```suggestion
    payload: { id },
```